plenum/test/node_request/test_node_got_no_preprepare.py

```diff
@@ -0,0 +1,129 @@
+import pytest
+
+from plenum.test.node_request.helper import nodes_last_ordered_equal
+from stp_core.loop.eventually import eventually
+
+from plenum.test.helper import sdk_send_batches_of_random_and_check, sdk_send_batches_of_random
+from plenum.test.malicious_behaviors_node import router_dont_accept_messages_from, reset_router_accepting
+
+from plenum.test.checkpoints.conftest import chkFreqPatched
+
+
+def tconf(tconf):
+    OldMax3PCBatchSize = tconf.Max3PCBatchSize
+    tconf.Max3PCBatchSize = 2
+    yield tconf
+    tconf.Max3PCBatchSize = OldMax3PCBatchSize
+
+
+def test_1_node_got_no_preprepare(looper,
+                                  txnPoolNodeSet,
+                                  sdk_pool_handle,
+                                  sdk_wallet_client,
+                                  tconf,
+                                  chkFreqPatched):
+    master_node = txnPoolNodeSet[0]
+    behind_node = txnPoolNodeSet[-1]
+    delta = tconf.CHK_FREQ * 3
```

Comment:
Please patch CHK_FREQ  (set CHK_FREQ = 3) in config for this test to reduce a number of batches that need to be sent

Reply:
I think that chkFreqPatched fixture do this

Reply:
Yes, sure. I didn't notice this.